user_guide/data_management.rst

```diff
@@ -0,0 +1,234 @@
+is one of the grid properties. 
+
+For example, you can open a grid of all the Contacts. (The Contact records represent actual people you are dealing with 
+and their properties are personal information, details of their position in the customer-company, contact details, etc.
+The contacts may be related to a customer, an opportunity, or any other record in the system, if reasonable.)
+
+So, we have opened the *"All Contacts"* grid. 
+
+.. image:: ./img/data_management/grid/grid_01.png
+
+We immediately see some major details of our contacts, such as their first and last names, email, phone number, etc.
+
+You can adjust and save grid views, updated and delete grid records, get to the Edit and View pages of any grid record, 
+and even export all the data in the grid as a .csv file.
+
+The grid details are described in the :ref:`Grids <user-guide-ui-components-grids>` guide.
+
+View Details of a Specific Record
+---------------------------------
+
+To see more details of a specific record, and to work directly with the record (create a task related to a customer, 
+appoint a calendar event for a user, turn a cart into an order, share a contact and so on), you need to get to its 
+View page.
+
+For example, we have opened the View page of one of our Contacts.
+
+.. image:: ./img/data_management/view/view_01.png
+
+On the View page you can see all the details of the record and location of the address on the map. You can 
+click the phone number link to call it via Hangouts or log a call, start writing an email with a click on the address, 
+initiate Skype session directly from the View page. If there is a related entity, its identifier will be a link to the 
+View page of this entity
+
+Action buttons at the top of each View page reflect the actions that you can do with the record. 
+
+The View Page details are described in the :ref:`View Pages <user-guide-ui-components-view-pages>` guide.
+
+Create New Records and Edit the Existing
+----------------------------------------
+
+To change details of and existing record or create a new record, you need to (re)define the record properties on its 
+Create/Edit form.
+
+The Create/Edit form details are described in the :ref:`Create/Edit Forms `<user-guide-ui-components-create-pages>` 
+guide.
+
+Tag the Records
+---------------
+
+In order to give the system users additional information about the record, to systematize the records and mark them for 
+future usage (e.g. to select them for reports or create segments), you can tag them with existing or new tags. For 
+example, if you tag a customer as "VIP", this will let the other system users know about the customer's importance, you 
+can adjust report to show only the data for your VIPs, and make targeted mailings to them. Moreover, you can find a 
+record by its tag with the :ref:`search <user-guide-getting-started-search>` functionality.
```

Comment:
Edit: In order to give system users additional information about a record, as well as to systematize records and mark them for future usage (e.g., to select them for reports or create segments), you can assign them existing or new tags. For example, if you tag a customer as "VIP," this will let the other system users know about the customer's importance. You can then adjust the report to show only data for your VIPs and make targeted mailings to them. Moreover, you can find a record by its tag with the :ref:`search <user-guide-getting-started-search>` functionality.